### ISSUE TITLE
"unrecognized artist" is clearly incorrect

### ISSUE THREAD

**Issue by Radioo** [kawaiiDango/pScrobbler#274]
![Screenshot_20221014-133106_Pano_Scrobbler](https://user-images.githubusercontent.com/9533160/195837121-2046ace6-c750-4240-81fa-b70b42e263e0.png)
![Screenshot_20221014-133039](https://user-images.githubusercontent.com/9533160/195837137-5205e7c9-ab3f-4d1d-bbfe-4abc27c23452.png)

**Comment by Radioo** [comment]
Yeah, it went through this time. I feel like this should become a setting like: "Ignore album artist".

**Comment by RuthlessRuler** [comment]
> Yeah, it went through this time. I feel like this should become a setting like: "Ignore album artist".

I think this is probably your Music Player App broadcasting this metadata as I don't have the option to enter/edit Album Artist.
